sending-email.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Sending email
+layout: recommendation
+expires: 2017-11-01
+---
+
+
+## User needs for sending email
+
+- applications notifying users about progress or actions to take
+- low level systems notifying engineers of failure if they cant be made to log errors
+
+## How to send email
+
+Applications notifying users should use [GOV.UK Notify](https://www.notifications.service.gov.uk/).
+Low level systems notifying engineers should use [Amazon SES](https://aws.amazon.com/ses/)
```

Comment:
Can we add a reference to the recommendation in https://alphagov.github.io/gds-tech/alerting.html#principles-for-setting-up-an-alerting-system of "Don’t rely on emails as an automated alerting method for incident responders".

I'm not sure "Low level" is very clear here.  I think we just mean "Automated notifications sent to engineers" - is that right?

What systems do we have currently that send such emails?